buildsystem/codecompliance/util.py

```diff
@@ -49,6 +49,26 @@ def readfile(filename):
     return FILECACHE[filename]
 
 
+def find_all_files(paths):
+    """
+    yields all files in paths.
+
+    hidden dirs and files are ignored.
+    """
+    for path in paths:
+        for filename in os.listdir(path):
+            if filename.startswith('.'):
+                continue
+
+            filename = os.path.join(path, filename)
+
+            if os.path.isdir(filename):
+                yield from find_all_files((filename,))
+                continue
+
+            yield filename
```

Comment:
I thought I could make it smaller with out the extension checking (turns out I cant) and then forgot about it... I will change it